backend/service/topology/cache.go

```diff
@@ -172,3 +173,32 @@ func (c *client) deleteCache(ctx context.Context, id string) error {
 	c.scope.SubScope("cache").Counter("delete.success").Inc(1)
 	return nil
 }
+
+func (c *client) expireCache(ctx context.Context) {
+	// Delete all entries that are older than two hours
+	const expireQuery = `
+		DELETE FROM topology_cache WHERE updated_at <= NOW() - INTERVAL '120m';
```

Comment:
nit: i'm sure this is minutes but the docs are terrible in trying to verify that. write out minutes in full please.